src/cms/templates/pois/poi_form.html

```diff
@@ -52,10 +52,10 @@ <h2 class="heading font-normal">
                             {% endif %}
                         {% endif %}
                     {% else %}
-                        {% trans 'Create new POI translation' %}
+                        {% trans 'Create new location' %}
```

Comment:
Is this correct?

Reply:
You're right, this is probably a mistake. I'll fix this in the merge commit together with the conflict resolving.